spellcast.py

```diff
@@ -130,35 +124,32 @@ def recalculate(self):
                 self.letter_multipliers[(row, column)],
             )
 
-        self.letter_values = {
+        letter_values = {
             letter: val * max_character_multiplier[letter]
             for letter, val in LETTERS_AND_VALUES.items()
         }
-        self.board_value = {}
-        for row, column in product(range(board_row_count), range(board_column_count)):
-            self.board_value[(row, column)] = self.letter_values[
-                self.board[row][column].lower()
-            ]
 
         long_word_bonus_points = 10
         long_word_minimum_letter_count = 6
-        self.calculate_value = lambda word: sum(
-            self.letter_values[character.lower()]
-            for character in word
-            if character.lower() in LETTERS_AND_VALUES
-        ) + (
-            long_word_bonus_points if len(word) > long_word_minimum_letter_count else 0
-        )
-
-        self.word_values = [(self.calculate_value(word), word) for word in self.words]
+
+        def calculate_value(word):
+            sum(

```

Comment:
This is missing `return`

Reply:
It was actually working as is but still the suggested change was applied.